src/index.js

```diff
@@ -1,36 +1,34 @@
-import React from 'react';
-import { render } from 'react-dom';
-import CompanyName from './views/create-company';
-import Chart from './views/chart';
-import CeoName from './views/ceo-view';
+import React from "react";
+import { render } from "react-dom";
+import CompanyName from "./views/create-company";
+import ChartView from "./views/chart";
+import CeoName from "./views/ceo-view";
```

Comment:
Great, this helps us to standardize the way we import some files. 
Here I will attach you a file to check more information: [click here](https://staxmanade.com/2018/03/should-i-use-javascript-single-or-double-quotes/)

Reply:
May be using editorconfig can help too.